packages/protocol/scripts/prepare-contracts-and-abis-publishing.ts

```diff
@@ -67,42 +68,23 @@ try {
   log('Running yarn wagmi generate')
   child_process.execSync(`yarn wagmi generate`, { stdio: 'inherit' })
 
-  // Generate an index.ts to be esm friendly
-  log('Generate index.ts from `CoreContracts`')
-  fs.writeFileSync(
-    path.join(ABIS_BUILD_DIR, 'index.ts'),
-    '// This is a generated file, do not modify\n' +
-      [...new Set(CoreContracts)]
-        .map((contract) => {
-          return `export * as ${contract} from './${contract}';`
-        })
-        .join('\n')
-  )
+  const sourcePackageJson = path.join(ABIS_PACKAGE_SRC_DIR, 'package.json.dist')
+  const destinationPackageJson = path.join(ABIS_BUILD_DIR, 'package.json')
 
-  // Generate the js folder to be published from ts files
-  log('Running tsc -b ')
-  child_process.execSync(`yarn tsc -b ${path.join(ABIS_PACKAGE_SRC_DIR, 'tsconfig.json')}`, {
-    stdio: 'inherit',
-  })
-  child_process.execSync(`yarn tsc -b ${path.join(ABIS_PACKAGE_SRC_DIR, 'tsconfig-cjs.json')}`, {
-    stdio: 'inherit',
-  })
+  fs.copyFileSync(
+    path.join(ABIS_PACKAGE_SRC_DIR, 'README.md'),
+    path.join(ABIS_BUILD_DIR, 'README.md')
+  )
 
   // Change the packages version to what CI is providing from environment variables
-  const packageJsons = [
-    path.join(CONTRACTS_PACKAGE_SRC_DIR, 'package.json'),
-    path.join(ABIS_PACKAGE_SRC_DIR, 'package.json'),
-  ]
-  packageJsons.forEach((packageJsonPath) => {
-    const file = fs.readFileSync(packageJsonPath).toString()
-
-    if (process.env.RELEASE_VERSION) {
-      fs.writeFileSync(
-        packageJsonPath,
-        file.replace('0.0.0-template.version', process.env.RELEASE_VERSION)
-      )
-    }
-  })
+  const file = fs.readFileSync(sourcePackageJson).toString()
+
+  if (process.env.RELEASE_VERSION) {
```

Comment:
doing some validity check on this would probably be wise. I think SemVer package is already used in the script?